payments-core/src/main/java/com/stripe/android/view/CardInputWidget.kt

```diff
@@ -585,7 +585,8 @@ class CardInputWidget @JvmOverloads constructor(
         return bundleOf(
             STATE_SUPER_STATE to super.onSaveInstanceState(),
             STATE_CARD_VIEWED to isShowingFullCard,
-            STATE_POSTAL_CODE_ENABLED to postalCodeEnabled
+            STATE_POSTAL_CODE_ENABLED to postalCodeEnabled,
+            //"HANDLE" to SavedStateHandle() // add tag so we can look up
```

Comment:
We should be able to handle this in the base layout. What's the consequences of not doing it now?